cli/main.go

```diff
@@ -213,7 +212,26 @@ func main() {
 		compose.Command(ctype),
 		volume.Command(ctype),
 	)
+	if ctype == store.DefaultContextType || ctype == store.LocalContextType {
+		if len(opts.Hosts) > 0 {
```

Comment:
why change the current context ? I'd rather error with an explicit message if users are changing the host and also specifying -H, if we don't support overriding the local context. 
And definitely error if context is ACI or ECS